app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,19 @@
+class AnswersController < ApplicationController
+    def create
+        @question = Question.find(params[:question_id])
+        @answer = @question.answer.new(answer_parmas)
+        @answer.question_id = @question
+        if @answer.save
+            flash[:notice]="回答を投稿しました"
+            redirect_to("/questions/#{params[:question_id]}")
```

Comment:
`@question` を定義しているので，詳細ページに戻る場合は次のようにかけますよ。

```suggestion
            redirect_to @question
```